python-usfm-parser/requirements.txt

```diff
@@ -1,2 +1,4 @@
 tree-sitter==0.20.0
-jupyterlab==3.3.2
+jupyterlab==3.4.4
```

Comment:
I think this is okay for now. But maybe you should move `jupyterlab` to a `dev-requirements.txt` file so that it doesn't get mixed up with the actual package dependencies.

Reply:
I am looking into how python modules are packaged for publishing on pypi. I think this file is not used there. But I shall make changes if required to separate it as dev dependency in a later PR